website_sale_charge_payment_fee/static/src/js/website_sale_fee.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function () {
```

Comment:
I'd switch this to snippet animation

Reply:
What is it? Do you have an example? Consider I followed the same approach of ```website_sale_delivery```

Reply:
quick example https://github.com/OCA/website-cms/blob/10.0/cms_form/static/src/js/lock_copy_paste.js
but if you search JS files for 'web_editor.snippets.animation' you'll find tons of examples.
Not mandatory at all, especially since is a migration :wink: